### ISSUE TITLE
Group RAW and JPEG as one item

### ISSUE THREAD

**Issue by davidak** [damonlynch/rapid-photo-downloader#107]
Many professional cameras allow to save a RAW + JPEG file for one photo. They often have the same name, but a different file extension. Other software (i think Lightroom and darktable) group them as one item and this software should too.

![Screenshot from 2023-06-10 13-38-51](https://github.com/damonlynch/rapid-photo-downloader/assets/91113/cd61c710-30a1-4e79-b83f-84cdcd232426)

This would lead to better UX, because when i unselect one picture, both files are unselected.

That would also make it faster to generate thumbnails, because it would be done for half the files.

**Comment by damonlynch** [comment]
Of course, I am familiar with the RAW + JPEG feature in cameras and what it does. If you look in the documentation and the file renaming preferences, that should be clear.

For this proposed change, you are welcome to write the code yourself. However, implement the proposed change as an option the user can turn on or off using the UI.

**Comment by obsidianeagle076** [comment]
+1 for this. You can just grab the thumbnails from the JPEG images to speed things up, and avoid compatibility issues like #315
